packages/kubecost/kubecost.patch

```diff
@@ -0,0 +1,11 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/kubecost/charts-original/Chart.yaml packages/kubecost/charts/Chart.yaml
+--- packages/kubecost/charts-original/Chart.yaml
++++ packages/kubecost/charts/Chart.yaml
+@@ -4,3 +4,7 @@
+   cloud costs.
+ name: cost-analyzer
+ version: 1.70.0
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/namespace: kubecost
```

Comment:
Please remove this annotation as this will prevent Rancher from handling the namespace for it. I will update our docs accordingly (I noticed we forgot to update it).

Reply:
Just made the change. It's also in the main-source branch README, so it should probably be updated there as well.